Homework 3/CloudObjectManager.js

```diff
@@ -50,37 +139,71 @@ class CloudObjectManager{
 			}
 		});
 	}
-	getProductsOfStore(store, callBack){
-		var query = new CB.CloudQuery("ProductStore");
-		query.equalTo('store_id', store);
-		query.find({
+	createStoreUserRelation(store, user, callBack){
+		var newRelation = new CB.CloudObject('StoreUser');
+		newRelation.set('user_id', user);
+		newRelation.set('store_id', store);
+		newRelation.save({
+			success: function(obj){
```

Comment:
Use ES6 function syntax.

```js
newRelation.save({
  success: (obj) => {
    callBack();
  },
  error: (err) => {
  }
});
```

This can actually be refactored more. I think you can actually call callback directly since it's function.

```js
newRelation.save({
  success: callback,
  error: (err) => { }
});
```

I believe you can actually do the following in ES6. It's shorthand to represent the same thing as above.

```js
newRelation.save({
  success(obj) {
    callBack();
  },
  error(err) {
  }
});
```